### PR TITLE
[CNXC-343] Display Series Description in dropdown for each study in PastAnalyses 

### PR DESCRIPTION
**Pull request by mslwang**
Showing image file names for studies in PastAnalyses isn't desirable behaviour. Instead, we want to have Series Descriptions for each image (shown below for example)
image
Replacing it in the same location
image

### CODE REVIEW

File: src/components/CreateAnalysis/ImageSelectionBox.tsx [hunk 1]
```diff
@@ -33,8 +33,8 @@ const ImageSelectionBox: React.FC<ImageSelectionBoxProps> = ({ img }) => {
           <img src={image_placeholder} alt="selected" width={60} height={60} />
           <div className="margin_left">
             <p className="color_grey">{StudyDescription}</p>
-            <p className="color_grey">Filename{fname.split('/')[3]}</p>
-            <p className="color_grey">{formatDate(creation_date)}</p>
+            <p className="color_grey">{"Filename: " + fname.split('/')[3]}</p>
```
Comment (mslwang) [inline src/components/CreateAnalysis/ImageSelectionBox.tsx:36]: Would it be better to move this commit to another PR? it's not wholly related, but I made this change in the process of walking through the process of solving this issue.

Comment (mslwang) [reply src/components/CreateAnalysis/ImageSelectionBox.tsx:36]: Before:
Screenshot from 2021-05-13 17-40-19
After:
Screenshot from 2021-05-13 17-36-32

**Comment by mslwang** [comment]
Before:
Screenshot from 2021-05-13 17-40-33

After:
Screenshot from 2021-05-13 17-34-25

**Review by micklynch — APPROVED** [review]
